app/views/helpers/FullResultsPageHelper.scala

```diff
@@ -145,7 +145,17 @@ object FullResultsPageHelper extends ViewHelper {
 
     val days = marginalRate.days
 
-    val upperLimit = CurrencyUtils.format(yearConfig.upperThreshold)
+    def upperLimit = CurrencyUtils.format(yearConfig.upperThreshold)
```

Comment:
Are these changes covered by tests?

Reply:
Whole FullResultsPageHelper is covered except for this line: throw new RuntimeException("Configuration is flat where it should be marginal")